src/components/FormGroup.jsx

```diff
@@ -1,17 +1,35 @@
 import React from 'react';
 
 const FormGroup = (props) => {
-  const { label, ...rest } = props
+  console.log(props);
+  const { label, ...rest } = props;
   const { styles } = FormGroup;
 
+  function inputField() {
+    if (props.placeholder === 'what are you looking for?') {
```

Comment:
I like the idea of making it conditional, but I feel like this could cause some issues if there is another input that requires black text in the future.  I could be reading it wrong.